documentation/multicut/multicut.md

```diff
@@ -71,7 +79,10 @@ Let's go throught the controls of this applet from top to bottom:
 
 ## Training and multicut
 Now that we have superpixels, we can combine them into objects. If your boundary probability maps are of high quality (e.g. they come from a neural network that was trained on very similar data), you can apply the Multicut partitioning directly on the maps. For this case, leave the "Train edge classifier" checkbox unchecked.
-If you are not fully satisfied with your boundary predictions, we can further improve them by training a Random Forest to predict good and bad edges. The general approach we use was first described in [this publication](https://link.springer.com/chapter/10.1007%2F978-3-642-33712-3_56). Briefly, given the superpixels computed in the previous step, we now compute features on the edges of adjacent superpixels. These features include, for example, the summed intensity of the edge and the minimal and maximal intensity along it, as well as statisitics of the probability map and of the intensity inside the superpixels ("Select Features" button brings up a dialog which lets you choose features). After the features are computed, we predict -- for every edge independently -- if this edge should be dropped or preserved to achieve a correct segmentation. The "naive" way to proceed would be to then only take the edges which are classified as "to preserve" and use those as the final segmentation. This, however, would lead to an inconsistent segmentation with dangling edges inside the objects. Instead, we formulate a so-called multicut problem, where special constraints ensure no dangling edges are present and all segmented objects are closed surfaces, while following the classifier preferences for which edges to keep. This problem is NP-hard in general, but this applet uses excellent approximate solvers to deliver a solution quickly.
+If you are not fully satisfied with your boundary predictions, we can further improve them by training a Random Forest to predict good and bad edges. The general approach we use was first described in [this publication](https://link.springer.com/chapter/10.1007%2F978-3-642-33712-3_56). Briefly, given the superpixels computed in the previous step, we now compute features on the edges of adjacent superpixels.
+These features include the shape of the superpixel boundaries, and intensity statistics computed on the superpixel area/volume as well as along the boundaries.
```

Comment:
```suggestion
These features include the shape of the superpixels, as well as intensity statistics computed along the superpixel boundaries and within the superpixel area/volume.
```

Reply:
hmmm, so it is really the shape of the boundaries that is considered, not the shape of the superpixels themselves. Do you think that's adding too much detail?

Reply:
I guess the initial phrasing was slightly confusing to me then 😁 Better?

Reply:
awww :) almost - would this make sense to you? It's just that it will either be areas (in case of 2D superpixels) or volumes (for 3D supervoxels :))

```suggestion
These features include the shape of the superpixels' boundaries, as well as intensity statistics of the boundaries, and areas (or volumes).
```

Reply:
Sounds good!